esp-hal/src/analog/adc/calibration/line.rs

```diff
@@ -60,7 +60,7 @@ where
         // Try get the reference point (Dout, Vin) from efuse
         // Dout means mean raw ADC value when specified Vin applied to input.
         let (code, mv) = ADCI::cal_code(atten)
-            .map(|code| (code, ADCI::cal_mv(atten)))
+            .map(|code| (code, ADCI::cal_mv(atten).unwrap_or_else(|| 1100)))
```

Comment:
I think I'd prefer 1100 to be a part of `cal_mv`, with a reference of where it comes from.

Reply:
I moved it all the way into the esph2 module, that way the trait stays unaffected and it should work as before.